plugins/genesys-gl32xx/fu-genesys-gl32xx-device.c

```diff
@@ -491,6 +491,8 @@ fu_genesys_gl32xx_device_ensure_cid(FuGenesysGl32xxDevice *self, GError **error)
 		return FALSE;
 
 	cid = fu_memread_uint32(data, G_BIG_ENDIAN);
+	if (cid == 0x0)
```

Comment:
Definitely no 0x0.
for GL3224 it is `BLOCK\VEN_05E3&DEV_0749&CID_FFFFFFFF` and for GL3232 -- `BLOCK\VEN_05E3&DEV_0764&CID_22FFFFFF`
Let me clarify numbers with Genesys team.

Reply:
I've used those now, and lets wait back to hear from Genesys too -- thanks for sending that email.